Source/Scene/Cesium3DTileStyle.js

```diff
@@ -8,7 +8,7 @@ define([
        './BooleanRegularExpression',
        './ColorRampExpression',
        './ColorMapExpression',
-       './ConstantColorExpression',
+       './LiteralColorExpression',
```

Comment:
Search and replace throughout this file (and then test in Sandcastle to double check).